package/ErrorFormatter.roc

```diff
@@ -67,6 +68,9 @@ format_arg_extract_err = \err ->
                 InvalidValue reason ->
                     "The value provided to $(option_display_name option) was not a valid $(option_type_name option): $(reason)"
 
+                InvalidUnicode ->
+                    "The value provided to $(option_display_name option) was not valid UTF-8."
```

Comment:
The err is `InvalidUnicode` which tactfully sidesteps the u8/u16 thing, but the message here says utf-8, should we "invalid unicode"?

Reply:
I should probably call this `InvalidUtf8`, actually. The idea is that an `Arg` is either a `U8` or a `U16` list, but `Str` can only be valid UTF-8. We try to convert either the `U8` list or the `U16` list to UTF-8, and if that fails this error is raised.